client/client.go

```diff
@@ -282,15 +250,27 @@ func (c *Client) withAccountIDRegionAndNamespace(accountID, region, namespace st
 	}
 }
 
+func validateRegions(logger hclog.Logger, regions []string) (bool, error) {
+	// validate regions values
+	for i, region := range regions {
+		if i != 0 && region == "*" {
+			return false, fmt.Errorf("region wildcard \"*\" is only supported as first argument")
+		}
+	}
+
+	var wildcardAllRegions bool
+	if len(regions) == 1 && regions[0] == "*" || len(regions) == 0 {
```

Comment:
I think it's better to have `()` around the first check just to be on the safe side

Reply:
fixed